cmd/loki/loki-local-config.yaml

```diff
@@ -27,10 +27,10 @@ schema_config:
 
 storage_config:
   boltdb:
-    directory: /loki/index
+    directory: /data/loki/index
```

Comment:
@slim-bean you mentioned something about this ? I think you're good with that change right ?

Reply:
this used to be /tmp a long time ago and got changed to /loki

/loki is annoying because many operating systems don't easily let you create this directory

I don't know if /data makes this any easier? my mac is dead at the moment, can you create /data (or does it exist) on mac?

Reply:
No so it's the same :)

```
 sudo mkdir /data
Password:
mkdir: /data: Read-only file system
```

Reply:
I would be favor of moving this back to `/tmp` if we were going to change it all

Reply:
@cyriltovena Did you run that inside the OSX docker hypervisor? By default that directory isn't shared with the host (see https://docs.docker.com/docker-for-mac/#file-sharing)

So do you want the consistency with the helm image or `/tmp` in this file?

Reply:
We are talking about running locally on mac, not in docker at all.  This is useful for us for testing/debugging etc, just run the processes and use the loki-local-config.yaml.

This broke however when it was changed to `/loki` some time ago because you can't make this directory on a mac (nor `/data`)

#1833 changed this apparently, however `/loki` is also kind of a poor choice, it's both unusable on mac and not linux FHS compliant.  

Given the sort of dual use of this file as both the config file for a container and also running outside of containers I'm not sure the best path forward here to be honest

I'm inclined to say just leave it as is for now and we should tackle this in a separate issue.

Reply:
Ok, I've removed the config file change